junit-runtime/src/main/scala/com/novocode/junit/JUnitFramework.scala

```diff
@@ -21,6 +21,22 @@ final class JUnitFramework extends Framework {
     f.runner(args, remoteArgs, testClassLoader)
   }
 
+  /** Scala Native specific: Creates a worker runner for a given run.
```

Comment:
Let's keep this long comment only on the definition of the abstract def in `sbt.testing.Framework`. Otherwise it's bound not to stay in sync. Scaladoc is smart enough to inherit the Scaladoc comment of overridden methods anyway.